src/colvar_arithmeticpath.h

```diff
@@ -76,6 +89,7 @@ void ArithmeticPathBase<element_type, scalar_type, path_type>::initialize(size_t
     frame_element_distances.resize(total_frames, p_element);
     frame_indexes.resize(total_frames);
     exponents.resize(total_frames, scalar_type());
+    exponents2.resize(total_frames, scalar_type());
```

Comment:
Is the second argument (the default content) useful here? If not, I'd rather simplify the code.

Reply:
Thanks, I have simplified the code in the latest commit.